<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PostADefectForm</class>
 <widget class="QDialog" name="PostADefectForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>513</width>
    <height>527</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PostADefectForm</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>491</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>SelectTopics</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>PleaseSelectTopic</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TopicDescription</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>351</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_TopicDescription">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>50</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>.....Text.....</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>491</width>
     <height>431</height>
    </rect>
   </property>
   <property name="title">
    <string>AddNewThread</string>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Subject/Title(Max 100 Characters)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Language</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>AssignedTo</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>471</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>ThreadDescription</string>
    </property>
    <widget class="QTextEdit" name="textEditThrreadDescription">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>451</width>
       <height>151</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>310</y>
      <width>481</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>AttachAFile</string>
    </property>
    <widget class="QTextEdit" name="textEditAttachFile">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>371</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonPostThread">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>70</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>PostThread</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_Close">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>70</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_Browse">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>20</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Browse</string>
     </property>
    </widget>
   </widget>
   <widget class="QComboBox" name="comboBoxLanguage">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>60</y>
      <width>291</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxAssignedTo">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>100</y>
      <width>291</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditTitle">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>19</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
